pages/index.html

```diff
@@ -2,7 +2,7 @@
 
 {% load i18n %}
 
-{% block title %}{% trans "Free Digital DJ MP3 Mixing Software" %}{% endblock %}
+{% block title %}{% trans "Free DJ Mixing Software App" %}{% endblock %}
```

Comment:
also change the data-text="..." below in the twitter link? and download.html's title could also use updating